docs/config/intro.md

```diff
@@ -0,0 +1,59 @@
+Configuration Files
+===================
+
+Forge has built in support for creating and managing mod configuration files. The purpose of a configuration file is to add flexibility and allow end users to customize their experience.
+
+Creating
+--------
+
+Create a dedicated class. Use the builder instance to define the configuration structure and data.
+
+```java
+private BooleanValue mySetting;
+
+public ServerConfig(ForgeConfigSpec.Builder builder) {
+  builder.comment("examplemod settings"); // Comment at the start of the file
+  
+  builder.push("Example Category"); // 'Enter' the Category
+  
+  mySetting = builder
+    .comment("Set this to true to enable something") // Comment on the setting
+    .translation("examplemod.exampleboolserver") // Language translation key
+    .define("exampleboolserver", false); // Setting key and default value
+
+  builder.pop(); // 'Exit' the category
+}
+```
+
+Registration
+------------
+
+Create the Pair instance.
```

Comment:
You should explain why this is a Pair and why we're making this instance. Something like:
`To register the configuration you'll first need to build it, for this you can use the ForgeConfigSpec.Builder#configure method. This will return a Pair object which stores both the configuration builder and your configuration file itself. Then you'll need to register this instance of your configuration file.`
The configure method isn't the only way to do this, you can also create a ForgeConfigSpec.Builder instance manually and build it manually.

Reply:
This is not a java tutorial. https://mcforge.readthedocs.io/en/1.14.x/styleguide/